geode-dunit/src/main/java/org/apache/geode/management/internal/cli/commands/QueryCommandDUnitTestBase.java

```diff
@@ -310,4 +323,40 @@ public void setValue1(Value1 value1) {
       this.value1 = value1;
     }
   }
+
+  @Test
+  public void testSimpleQueryWithProxyRegion() {
```

Comment:
Would it make sense to add the same test for a `PARTITION_PROXY` region?, or the changes within this `PR` doesn't affect that use case?.

Reply:
It does not affect PR.